presto-spi/src/main/java/com/facebook/presto/spi/security/AccessControlContext.java

```diff
@@ -48,6 +51,11 @@ public Optional<String> getClientInfo()
         return clientInfo;
     }
 
+    public Set<String> getClientTags()
```

Comment:
exposes potentially mutable internal state. Might be risky in a security context.